topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -21,16 +21,20 @@ After a corrected Hi-C matrix is created other tools can be used to visualize it
 >
 > In this tutorial, we will deal with:
 >
-> 1. Mapping Hi-C data
-> 2. Create a contact matrix
-> 3. Learn how to correct a contact matrix
-> 4. Visualize a contact matrix
-> 5. Compute and visualize topological associating domains (TADs)
+> 1. Data upload
+> 2. Mapping Hi-C data
+> 3. Create a contact matrix
+> 4. Learn how to correct a contact matrix
+> 5. Visualize a contact matrix
+> 6. Compute and visualize topological associating domains (TADs)

```

Comment:
you do not need to make the entries manually, can replace the entire agenda box with the following and it will automatically generate it based on the section names :)